lib/stock_markets/markets.rb

```diff
@@ -0,0 +1,43 @@
+module StockMarkets
+  class Markets
+    extend Enumerable
+
+    class << self
+      def each(&block)
+        stock_markets_with_mics.values.each do |stock|
+          block.call(stock)
+        end
+      end
+
+      def [](stock_mic)
```

Comment:
why did you remove lookup by `mic_or_name`?

Reply:
Discussed. Resolved